geotorch/hurwitz.py

```diff
@@ -0,0 +1,164 @@
+import torch
+from .psd import PSD
+from .skew import Skew
+from .product import ProductManifold
+from .exceptions import VectorError, NonSquareError
+from .utils import _extra_repr
+from .exceptions import (
+    VectorError,
+    NonSquareError,
+    InManifoldError,
+)
+
+def get_lyap_exp(A):
+    return -torch.max(torch.real(torch.linalg.eigvals(A)))
+
+class Hurwitz(ProductManifold):
+
+    def __init__(self, size, alpha: float = 0.0, triv="expm"):
+        r"""
+        Manifold of matrices with eigenvalues with negative real parts,
+        also called Hurwitz matrices. They represend linear stable linear dynamical systems.
```

Comment:
```suggestion
        also called Hurwitz matrices.
```
no need to discuss applications in the docs.